.circleci/config.yml

```diff
@@ -0,0 +1,37 @@
+version: 2
+jobs:
+  lint:
+    docker:
+      - image: circleci/python:2.7.18
+    steps:
+      - run:
+          name: lint code
+          command: |
+            pip install --upgrade pip
+            pip install flake8
+            flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
```

Comment:
Let's add a Makefile with this command so that developers can run linting locally.

Reply:
Instead of using --select, can we just use the default flake8 checks?

Reply:
@adborden created #44 to use a Makefile.

Using the default flake8 check [we have 362 errors](https://app.circleci.com/pipelines/github/GSA/USMetadata/4/workflows/44576b77-a888-47bc-ac77-5087c485bee4/jobs/5)

Do we want to fix them?

```
2     E101 indentation contains mixed spaces and tabs
3     E124 closing bracket does not match visual indentation
3     E127 continuation line over-indented for visual indent
28    E128 continuation line under-indented for visual indent
1     E129 visually indented line with same indent as next logical line
1     E201 whitespace after '{'
2     E225 missing whitespace around operator
8     E231 missing whitespace after ','
13    E251 unexpected spaces around keyword / parameter equals
1     E261 at least two spaces before inline comment
5     E265 block comment should start with '# '
1     E301 expected 1 blank line, found 0
3     E302 expected 2 blank lines, found 1
1     E303 too many blank lines (2)
1     E402 module level import not at top of file
254   E501 line too long (91 > 79 characters)
10    F401 'sqlalchemy.MetaData' imported but unused
2     F402 import 'v' from line 8 shadowed by loop variable
1     F841 local variable 'q' is assigned to but never used
18    W191 indentation contains tabs
2     W292 no newline at end of file
2     W603 '<>' is deprecated, use '!='
362
```

Reply:
Most are "line too long". Maybe exclude that rule (120 char line length is better anyway) and fix the others?